sql-to-dbsp-compiler/lib/sqllib/src/operators.rs

```diff
@@ -313,3 +313,12 @@ pub fn div_dN_dN(left: Option<F64>, right: Option<F64>) -> Option<F64> {
 pub fn plus_u_u(left: usize, right: usize) -> usize {
     left + right
 }
+
+#[inline(always)]
```

Comment:
you need many more functions here, for all types that are supported by safe_add, following the example of `plus`.
you probably also need some calls to the macro `for_all_int_operators!`

Reply:
The documentation is also missing.